dragonchain/lib/broadcast.py

```diff
@@ -36,6 +37,8 @@
 
 def dispatch(block: "model.BlockModel") -> None:
     if LEVEL == "5":
+        _log.info("Trying to finalize any previously failed claims...")
+        process_claims_backlog()
```

Comment:
it doesn't make sense to even have this here. This will only process a backlog every time an L5 actually transmits a block. That means it might only retry once every 2 or 3 hours. This should be retried every time the transaction processor runs, not just when it happens to be broadcasting a block

Reply:
Moved function to `level_5_actions`, makes more sense for it to belong there and be invoked there.